src/Microsoft.DotNet.XHarness.CLI/Commands/WASM/WasmTestMessagesProcessor.cs

```diff
@@ -133,7 +136,14 @@ private void InvokeInternal(string message)
                     catch (Exception ex)
                     {
                         _logger.LogError($"Error while saving testResults.xml {ex}");
-                        throw;
```

Comment:
Shouldn't we process this way only XML related exceptions? If we really want catch all exceptions this way, I would probably name the property differently.